PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -1338,13 +1421,17 @@ - (PNRequestParameters *)subscribeRequestParametersWithState:(NSDictionary *)sta
                              forFieldName:@"state"];
         }
     }
+    if (self.client.configuration.filterExpression) {
+        [parameters addQueryParameter:[self escapedFilterExpression] forFieldName:@"filter-expr"];
+    }
```

Comment:
@jzucker2 I see you check whether user specified filter expression or not during client confirmation directly using _PNConfiguration_ property. I think we can remove publicly exposed (in PubNub+Core) _-filterExpression_ method (there is no API requirements to expose it).